android-components/config/detekt.yml

```diff
@@ -32,7 +39,15 @@ console-reports:
   #  - 'NotificationReport'
   #  - 'FindingsReport'
   #  - 'FileBasedFindingsReport'
-    - 'LiteFindingsReport'
+  #  - 'LiteFindingsReport'
```

Comment:
Uncommented this since there's no reason to really exclude this or any particular reports.